masterlist.yaml

```
@@ -28190,3 +28190,59 @@ plugins:
         crc: 0x9E8F3CA5
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 17
+
+  - name: 'UnforgivingDevices - DD Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices DD Patch'
+    after:
+      - 'Devious Devices SE patch.esp'
+      - 'Devious Devices - Lore Patch.esp'
+    clean:
+      - crc: 0xD32A2D77
+        util: 'SSEEdit v4.0.4'
+  - name: 'UnforgivingDevices - LBS Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices Lauras Bondage Shop Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
```

Single-element lists should be written in flow style:

```suggestion
    after: [ 'UnforgivingDevices - DD Patch.esp' ]
```